apps/src/netsim/NetSimTable.js

```diff
@@ -230,18 +269,43 @@ NetSimTable.prototype.synchronousDelete = function (id) {
  */
 NetSimTable.prototype.fullCacheUpdate_ = function (allRows) {
   // Rebuild entire cache
+  var maxRowID = 0;
   var newCache = allRows.reduce(function (prev, currentRow) {
     prev[currentRow.id] = currentRow;
+    if (currentRow.id > maxRowID) {
+      maxRowID = currentRow.id;
+    }
     return prev;
   }, {});
 
   // Check for changes, if anything changed notify all observers on table.
   if (!_.isEqual(this.cache_, newCache)) {
     this.cache_ = newCache;
+    this.latestRowID_ = maxRowID;
+    this.tableChange.notifyObservers(this.arrayFromCache_());
+  }
+
+  this.lastRefreshTime_ = Date.now();
+};
+
+/**
+ * @param {Array} newRows
```

Comment:
Please add documentation explaining what this function does.  (It's fine to document @private methods.)